src/player.h

```diff
@@ -594,6 +594,7 @@ class Player final : public Creature, public Cylinder
 		bool isImmune(ConditionType_t type) const final;
 		bool hasShield() const;
 		bool isAttackable() const final;
+		bool lastHitIsPlayer(Creature* lastHitCreature) const;
```

Comment:
Use a free function, not a method for faster compiling as it's only useful in player.cpp compilation unit (will never be used outside)